### PR TITLE
[Amplitude] Send Record on Teacher Login

### PR DESCRIPTION
**Pull request by epeach**
Part of our Amplitude project.
Triggers when a teacher renders the homepage, but only once per session (to emulate the 'on login' experience).

Includes testing.

### CODE REVIEW

File: apps/src/templates/studioHomepages/TeacherHomepage.jsx [hunk 1]
```diff
@@ -149,6 +155,19 @@ export const UnconnectedTeacherHomepage = ({
...
 
+  // Send one analytics event when a teacher logs in. Use session storage to determine
+  // whether they've just logged in.
+  if (
+    !!currentUserId &&
+    tryGetLocalStorage(LOGGED_TEACHER_SESSION, 'false') !== 'true'
+  ) {
+    trySetLocalStorage(LOGGED_TEACHER_SESSION, 'true');
+
+    analyticsReporter.sendEvent(EVENTS.TEACHER_LOGIN_EVENT, {
```
Comment (bethanyaconnor) [inline apps/src/templates/studioHomepages/TeacherHomepage.jsx:166]: Forgive my ignorance on local storage -- would this trigger correctly if a teacher logged in, logged out, and logged in again?

File: apps/test/unit/templates/studioHomepages/StudentHomepageTest.js [hunk 2]
```diff
@@ -57,6 +60,13 @@ describe('StudentHomepage', () => {
     });
   });
 
+  it('does not log an Amplitude event for teacher signing-in', () => {
```
Comment (bethanyaconnor) [inline apps/test/unit/templates/studioHomepages/StudentHomepageTest.js]: nit: should this test name be "does not log an Amplitude event for **student** signing-in"?

File: apps/src/templates/studioHomepages/TeacherHomepage.jsx (hunk 1 above)
Comment (epeach) [reply apps/src/templates/studioHomepages/TeacherHomepage.jsx:166]: Good catch! I meant for this to be `trySetSESSIONStorage` - fixing!

**Review by bethanyaconnor — APPROVED** [review]
Thanks for adding tests!

**Pull request merged**